### DISCUSSION TITLE
How do I change the anchor point of a tooltip?

### DISCUSSION THREAD

**Discussion by iamkd** [airbnb/visx#1701] [General]
Hi! I've been trying to make a tooltip display to the left of the pointer. See https://codesandbox.io/s/suspicious-faraday-3mtds. I've tried adding the offset (-200px) to the `tooltipLeft` parameter, but it does not detect bounds correctly. How can I fix that?

I'd really like to be able to use <TooltipWithBounds /> and specify the anchor point of a tooltip (top-left, bottom-right, center-left etc.) Thanks!

**Comment by MYKEU** [comment]
I guess there is no real solution for this
